packages/utils/text.tsx

```diff
@@ -34,4 +34,10 @@ export const replaceBetweenString = (
   `${origin.substring(0, startIndex)}${insertion}${origin.substring(endIndex)}`;
 
 export const pluralOrNot = (word: string, quantity: number) =>
```

Comment:
I think for most of cases , this is fine if we don't need the perfect solution :D 
But if we are perfectionist, we have to handle many edge cases like words finished by s (like gas) or ch ...

Reply:
Yes, I added the lib `pluralize` => https://classic.yarnpkg.com/en/package/pluralize